### ISSUE TITLE
Trap focus inside of sidebar when it's open

### ISSUE THREAD

**Issue by meganesu** [meganesu/generate-gb-opcodes#17]
Currently, keyboard users can tab to the DetailsSidebar close button even when the sidebar is closed. Instead, the tabbable content inside of DetailsSidebar should be removed from the tab order when the sidebar is not open.

GIVEN the sidebar is closed
WHEN I press the tab key repeatedly
THEN I should not be able to access the tabbable content inside the sidebar

**Comment by meganesu** [comment]
See this a11ycast video for more information on this issue: https://www.youtube.com/watch?v=fGLp_gfMMGU&list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g&index=2
